docs/usage.md

```diff
@@ -207,105 +255,326 @@ which are optional:
+  image built from a recipe provided by a product.
+
+### Example bootprep input files
+
+This section provides an example bootprep input file and instructions for
+obtaining the default bootprep input files delivered with a release of the HPC
+CSM Software Recipe.
+
+#### Example bootprep input file
+
+The following bootprep input file provides an example of using most of the
+features described in previous sections. It is not intended to be a complete
+bootprep file for the entire CSM product.
+
+```yaml
 ---
 configurations:
-- name: cos-config
+- name: compute-{{recipe.version}}
   layers:
-  - name: cos-integration-2.2.87
+  - name: cos-compute-integration-{{cos.version}}
     playbook: site.yml
     product:
       name: cos
-      version: 2.2.87
-      branch: integration
-  - name: cpe-integration-21.12.3
+      version: "{{cos.version}}"
+      branch: integration-{{cos.version}}
+  - name: cpe-pe_deploy-integration-{{cpe.version}}
     playbook: pe_deploy.yml
     product:
       name: cpe
-      version: 21.12.3
-      branch: integration
-  - name: slurm-master-1.1.1
-    playbook: site.yml
-    product:
-      name: slurm
-      version: 1.1.1
-      branch: master
+      version: "{{cpe.version}}"
+      branch: integration-{{cpe.version}}
+
 images:
-- name: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  ims:
-    is_recipe: true
-    name: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  configuration: cos-config
+- name: "{{base.name}}"
+  ref_name: base_cos_image
+  base:
+    product:
+      name: cos
+      type: recipe
+      version: "{{cos.version}}"
+
+- name: compute-{{base.name}}
+  ref_name: compute_image
+  base:
+    image_ref: base_cos_image
+  configuration: compute-{{recipe.version}}
   configuration_group_names:
   - Compute
+
 session_templates:
-- name: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  image: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  configuration: cos-config
+- name: compute-{{recipe.version}}
+  image:
+    image_ref: compute_image
+  configuration: compute-{{recipe.version}}
   bos_parameters:
     boot_sets:
       compute:
         kernel_parameters: ip=dhcp quiet spire_join_token=${SPIRE_JOIN_TOKEN}
         node_roles_groups:
         - Compute
+        rootfs_provider_passthrough: "dvs:api-gw-service-nmn.local:300:hsn0,nmn0:0"
 ```
 
-#### Creating a pre-populated example bootprep input file
+#### Accessing default bootprep input files
+
+Default bootprep input files are delivered by the HPC CSM Software Recipe
+product. These files can be accessed by cloning the `hpc-csm-software-recipe`
+repository.
 
-It is possible to create an example bootprep input file using values from the
-system's product catalog using the `sat bootprep generate-example` command.
+Follow steps 1-7 of the procedure in [Viewing HPC CSM Software Recipe
+variables](#viewing-hpc-csm-software-recipe-variables).
+
+Access the files in the `bootprep` directory of that repository:
```

Comment:
I would consider duplicating steps 1-7 here and adding accessing the `bootprep` directory as step 8. For new users, this file can largely be read from top to bottom. However, we'll break that flow if we don't include all the steps under this heading. With a large part of the information in this file new for Papaya, I think we want to avoid that.

Reply:
I really want to avoid duplicating info and having to maintain it in two places.

There's also duplicate information in the Getting Started Guide for this, and I think that will be the more visible location for this information.

Eventually, I would like to add a `sat bootprep` subcommand that outputs the variables available from the HPC CSM Software Recipe, so that other procedure would not be necessary either. See [CRAYSAT-1532](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1532) for that.

Reply:
I'm fine not duplicating the instructions until we have an idea of the repo setup with unified docs. However, I'd suggest changing the wording a little (see next comment).